test/minikube/minikube_rolling_upgrade_test.go

```diff
@@ -74,9 +73,10 @@ func TestKubernetesUpgradeFullDatabaseMinorVersion(t *testing.T) {
 
 	options := helm.Options{
 		SetValues: map[string]string{
-			"nuodb.image.registry": "docker.io",
+			"nuodb.image.registry":   "docker.io",
```

Comment:
what editor are you using? I am guessing that we have different settings for intents.

Reply:
[Spacemacs](https://github.com/syl20bnr/spacemacs) with the [Golang layer](https://develop.spacemacs.org/layers/+lang/go/README.html).